### ISSUE TITLE
Regions are obsolete

### ISSUE THREAD

**Issue by RobinBressan** [jpetitcolas/french-geography#11]
I am confused, I can find the new regions like `Grand Est` into the data 😢

**Comment by RobinBressan** [comment]
Any news on this @jpetitcolas? This is a major drawback for production usage...

**Comment by jpetitcolas** [comment]
Feel free to open a PR! :trollface:

**Comment by jdemangeon** [comment]
Yes! @RobinBressan can you open a PR please? I really need this awesome feature

**Comment by RobinBressan** [comment]
You mean a Police Report? Let me find their phone number...

**Comment by RobinBressan** [comment]
any news @jpetitcolas?

**Comment by jpetitcolas** [comment]
![image](https://user-images.githubusercontent.com/688373/71251124-af4a5a80-2321-11ea-93b9-01eaf298a992.png)
